### ISSUE TITLE
Small-C support for static keyword

### ISSUE THREAD

**Issue by ZaneDubya** [ZaneDubya/SmallC#5]
This issue tracks the addition of the static keyword to Small-C. Definitions from https://en.wikipedia.org/wiki/Static_(keyword)

# Preparation
- [x] Change all uses of 'static' in Small-C compiler to 'global'. Static means something else.
- [x] Figure out where the 'static' (internal?) flag will be stored.

# Static global variables
A variable declared as static at the top level of a source file (outside any function definitions) is only visible throughout that file ("file scope", also known as "internal linkage"). In this usage, the keyword static is known as an "access specifier".
- [x] Add new amatch for static in parse
- [x] Add sanity ('static' only applies to variables and functions, can't do extern static, others?)
- [x] Don't export static global variables

# Static functions
Similarly, a static function -- a function declared as static at the top level of a source file (outside any class definitions) -- is only visible throughout that file ("file scope", also known as "internal linkage").
- [x] Don't export static functions

**Comment by ZaneDubya** [comment]
Static global and static functions done in 73e982f164db65800758d79ca98b55ea7278aa1f

**Comment by ZaneDubya** [comment]
Moved static local variables to #8 because (1) this is a separate issue, (2) it's blocked by #7. Closing issue!
